### PR TITLE
Update foreman_ansible to 1.3.1 [deb]

### PR DESCRIPTION
**Pull request by dLobatog**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.14
* [x] 1.13
* [ ] 1.12

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

Thanks!

### CODE REVIEW

**Comment by mmoll** [comment]
As I ran into this and forgot about it: Since app/assets/images/Ansible.png was added, this plugin does need asset precompilation.

**Comment by dLobatog** [comment]
Updated, thanks @mmoll 😄

File: plugins/ruby-foreman-ansible/debian/control [hunk 1]
```diff
@@ -12,6 +12,6 @@ Package: ruby-foreman-ansible
 Architecture: all
 Depends: ${misc:Depends}, bundler, foreman (>= 1.12.0~rc1),
   ruby-foreman-deface (<<2.0.0), ruby-foreman-tasks (>= 0.8.2-2),
-  ruby-foreman-tasks (<<0.9.0)
+  ruby-foreman-tasks (<<0.9.0), foreman-assets
```
Comment (mmoll) [inline plugins/ruby-foreman-ansible/debian/control]: foreman-assets also needs the foreman version

File: plugins/ruby-foreman-ansible/debian/install [hunk 2]
```diff
@@ -1,3 +1,4 @@
 foreman_ansible.rb  usr/share/foreman/bundler.d
 cache                usr/share/foreman/vendor
+assets              var/lib/foreman/public
```
Comment (mmoll) [inline plugins/ruby-foreman-ansible/debian/install]: indentation is off here

File: plugins/ruby-foreman-ansible/debian/rules [hunk 3]
```diff
@@ -17,10 +17,12 @@ build:
 	cd /usr/share/foreman && ( \
 		/usr/bin/foreman-ruby /usr/bin/bundle install --local && \
 		/usr/bin/foreman-ruby /usr/bin/bundle exec rake db:migrate RAILS_ENV=development && \
+		/usr/bin/foreman-ruby /usr/bin/bundle exec rake plugin:assets:precompile[foreman_ansible] RAILS_ENV=production && \
```
Comment (mmoll) [inline plugins/ruby-foreman-ansible/debian/rules]: `$PLUGIN` should be used instead of `foreman_ansible` hardcoded here

**Comment by mmoll** [comment]
The build currently fails, probably this is fallout from the current situation with foreman-tasks in core and packaging.

**Comment by mmoll** [comment]
superseded by #1498.

**Pull request closed without merging**